next.config.js

```diff
@@ -1,4 +1,16 @@
-module.exports = {
+const { withSentryConfig } = require('@sentry/nextjs')
+
+const moduleExports = {
   distDir: 'build/_next',
   target: 'server',
 }
+
+const { NODE_ENV, SENTRY_RELEASE } = process.env
+
+const sentryWebpackPluginOptions = {
+  dryRun: !(NODE_ENV === 'production'),
+  release: SENTRY_RELEASE,
+  silent: !(NODE_ENV === 'production'),
+}
```

Comment:
The Sentry webpack plugin tries to upload the source maps and create a release every time a build is created - not what you want when you're running e2e tests 🤦🏻‍♂️